test_files/clutz.no_externs/strip_clutz_type.js

```diff
@@ -1,4 +1,5 @@
 goog.module('test_files.clutz.no_externs.strip_clutz_type');var module = module || {id: 'test_files/clutz.no_externs/strip_clutz_type.js'};
+Object.defineProperty(exports, "__esModule", { value: true });
```

Comment:
From the above link you can go to the TS bug where they say:
"The behavior was there for modules with export default. now it is for all modules."
So I think google3 was ok because we disallow export default.